packages/editor/src/browser/editor-decoration.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
```

Comment:
`2018`, please check other copyrights in new files

Reply:
thanks!